### PR TITLE
Chatpage

### PR DESCRIPTION
**Pull request by smartdev0809**

### CODE REVIEW

File: src/pages/chat/message.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import LeftSide from "./leftside";
+import RightSide from "./rightSide";
+import MainSide from "./mainSide";
+import { useSelector } from "react-redux";
+
```
Comment (pandablue0809) [inline src/pages/chat/message.jsx]: plz remove this hook

File: src/pages/chat/mainSide.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  },
+  {
+    avatar: avatar3,
+    reporter: "Gru Tomato",
+    time: "3:12 5-12-2024",
+    content:
+      "Okay. Anyway the meeting will start at 8:00 AM. I am wery happy if you remember it.",
+    meReport: true,
+  },
+];
```
Comment (dot425) [inline src/pages/chat/mainSide.jsx:74]: plz make mokup data

File: src/pages/chat/leftside.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                  alt={user.firstName}
+                  className="rounded-full  w-12 h-12 border-2 border-white"
+                />
+              </div>
+              <div>
+                <span className="text-[#024754] text-lg ml-[-12px]">
+                  {user.lastName + " " + user.firstName}
+                </span>
+                <p className="text-gray-400 text-md pt-2 text-sm">
+                  {user.time}
```
Comment (charmhorse) [inline src/pages/chat/leftside.jsx:86]: add bio

**Comment by charmhorse** [comment]
LGTM

**Comment by dot425** [comment]
LGTM

**Comment by pandablue0809** [comment]
LGTM

**Pull request merged**